docs/tutorials/03_quantum_kernel.ipynb

```diff
@@ -342,44 +471,50 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We again set up the `FidelityQuantumKernel` class to calculate a kernel matrix using the [ZZFeatureMap](https://qiskit.org/documentation/stubs/qiskit.circuit.library.ZZFeatureMap.html), and the default values this time."
+    "## 2.1 Quantum Kernel\n",
+    "We again set up the `FidelityQuantumKernel` class to calculate a kernel matrix using the [ZZFeatureMap](https://qiskit.org/documentation/stubs/qiskit.circuit.library.ZZFeatureMap.html), and the default values this time. Revisit the [classification](#Classification) section for refreshing your concepts."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 9,
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Feature map\n",
     "adhoc_feature_map = ZZFeatureMap(feature_dimension=adhoc_dimension, reps=2, entanglement=\"linear\")\n",
-    "\n",
+    "# Kernel\n",
     "adhoc_kernel = FidelityQuantumKernel(feature_map=adhoc_feature_map)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The scikit-learn spectral clustering algorithm allows us to define a custom kernel in two ways: by providing the kernel as a callable function or by precomputing the kernel matrix. Using the `FidelityQuantumKernel` class in Qiskit Machine Learning, we can only use the latter.\n",
+    "## 2.2 Spectral Clustering Model\n",
+    "\n",
+    "The scikit-learn spectral clustering algorithm allows us to define a custom kernel in two ways (just like SVC):\n",
+    "* by providing the kernel as a callable function\n",
+    "* by precomputing the kernel matrix. \n",
+    "\n",
+    "For implementation of the `FidelityQuantumKernel` class in Qiskit Machine Learning, we can only use the latter option of kernel definition.\n",
     "\n",
     "The following code precomputes and plots the kernel matrices before providing it to the scikit-learn spectral clustering algorithm, and scoring the labels using normalized mutual information, since we a priori know the class labels."
```

Comment:
```suggestion
    "The following code precomputes and plots the kernel matrices before providing them to the scikit-learn spectral clustering algorithm, and scoring the labels using normalized mutual information, since we a priori know the class labels."
```